subjects/devops/string_tokenizer/README.md

```diff
@@ -0,0 +1,91 @@
+## String tokenizer
+
+### Instructions
+
+Tokenization is the process of breaking down a string into smaller pieces, called tokens. In natural language processing, tokenization typically refers to the process of breaking down a sentence into words or breaking down a paragraph into sentences.
+
+Create a file `string_processing.py` which will have a function `tokenize(sentence)` that given a sentence will do the following:
+
+- removes all punctuation marks and special characters
+- separates all words like so: `"it's not 3" => ['it', 's', 'not', '3']`
+- put all the words in lowercase
+- return a list of all the words.
+
+### Usage
+
+Here is a possible `test.py` to test your functions:
+
+```python
+import string_processing
+
+if __name__ == '__main__':
+    my_sentence = "It's not possible, you can't ask for a raise"
+    print(string_processing.tokenize(my_sentence))
+```
+
+```bash
+$ python test.py
+['it', 's', 'not', 'possible', 'you', 'can', 't', 'ask', 'for', 'a', 'raise']
+```
+
+### Hints
+
+The `re` library is a module for working with regular expressions it provides a set of functions for working with regular expressions, including:
```

Comment:
it's missing either a `,` or a `.` after expressions

Reply:
done